index.bs

```diff
@@ -5760,11 +5760,15 @@ be removed in a future revision of this specification. Its use is expected to be
 replaced by <code>initiatorType</code> and <code>destination</code> on
 <code>network.RequestData</code>.
 
+Note: The <code>request</code> field is included in the definition for backwards
+compatibility, but is no longer set by the [=get the initiator=] steps, and will
+be removed in a future revision of this specification. The network.Initiator is
+included in the BeforeRequestSentParameters which also contain the same request
+id, which makes the information redundant.
```

Comment:
Should we maybe add the reference to `BaseParameters` here to make it easier to find?

Reply:
Done